src/Stratis.Bitcoin.IntegrationTests/APITests.cs

```diff
@@ -89,40 +66,35 @@ public void CanGetGeneralInfoViaAPI()
         [Fact]
         public void CanStartStakingViaAPI()
         {
-            try
+            using (NodeBuilder builder = NodeBuilder.Create())
             {
-                var fullNode = this.apiTestsFixture.stratisStakeNode.FullNode;
-                var apiURI = fullNode.NodeService<ApiSettings>().ApiUri;
+                var posNode = builder.CreateStratisPosApiNode();
+                posNode.Start();
 
-                Assert.NotNull(fullNode.NodeService<IPosMinting>(true));
+                var apiURI = posNode.FullNode.NodeService<ApiSettings>().ApiUri;
 
-                using (client = new HttpClient())
+                using (var httpClient = new HttpClient())
                 {
-                    WalletManager walletManager = fullNode.NodeService<IWalletManager>() as WalletManager;
+                    var walletManager = posNode.FullNode.NodeService<IWalletManager>() as WalletManager;
 
                     // create the wallet
                     var model = new StartStakingRequest { Name = "apitest", Password = "123456" };
                     var mnemonic = walletManager.CreateWallet(model.Password, model.Name);
 
                     var content = new StringContent(model.ToString(), Encoding.UTF8, "application/json");
-                    var response = client.PostAsync(apiURI + "api/miner/startstaking", content).GetAwaiter().GetResult();
+                    var response = httpClient.PostAsync(apiURI + "api/miner/startstaking", content).GetAwaiter().GetResult();
                     Assert.Equal(System.Net.HttpStatusCode.OK, response.StatusCode);
 
                     var responseText = response.Content.ReadAsStringAsync().GetAwaiter().GetResult();
                     Assert.Equal("", responseText);
 
-                    MiningRPCController controller = fullNode.NodeService<MiningRPCController>();
+                    MiningRPCController controller = posNode.FullNode.NodeService<MiningRPCController>();
                     GetStakingInfoModel info = controller.GetStakingInfo();
```

Comment:
Lines above 91 you declare variables with the var key word.  Here you use typed variables - do you think it might look better to keep to be consistent throughout the method ? 

Even reduce down the lines by getting rid of `controller` variable 

![image](https://user-images.githubusercontent.com/16000683/38772023-58228434-4025-11e8-986e-73bbddd9f903.png)

Reply:
Fixed